tests/Data/Entries/CollectionTest.php

```diff
@@ -691,6 +691,47 @@ public function it_gets_preview_targets()
         $this->assertEquals([
             ['label' => 'Foo', 'format' => '{foo}'],
             ['label' => 'Bar', 'format' => '{bar}'],
+        ], $collection->basePreviewTargets()->all());
+    }
+
+    /** @test */
+    public function it_gets_extra_preview_targets()
+    {
+        $collection = (new Collection)->handle('test');
+
+        $this->assertInstanceOf(\Illuminate\Support\Collection::class, $collection->extraPreviewTargets());
+
+        $this->assertEquals([], $collection->extraPreviewTargets()->all());
+
```

Comment:
Probably a good idea to add a base preview target as extra assurance that extraPreviewTargets doesn't include it.